packages/protocol/contracts/L1/libs/proofTypes/LibVerifySGX.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+//  _____     _ _         _         _
+// |_   _|_ _(_) |_____  | |   __ _| |__ ___
+//   | |/ _` | | / / _ \ | |__/ _` | '_ (_-<
+//   |_|\__,_|_|_\_\___/ |____\__,_|_.__/__/
+
+pragma solidity ^0.8.18;
+
+import {AddressResolver} from "../../../common/AddressResolver.sol";
+import {LibUtils} from "../LibUtils.sol";
+import {TaikoData} from "../../TaikoData.sol";
+
+library LibVerifySGX {
+    error L1_INVALID_SGX_SIGNATURE();
+
+    function verifySgxProof(
```

Comment:
I think we can also use this library for non-sgx signers, so I suggested to change the name to LibVerifyTrusted